examples/testify/src/main/resources/application-test.yml

```diff
@@ -0,0 +1,11 @@
+spring.transaction.default-timeout: 60
+
+# Longer query timeout for specific users.
+yakworks:
+  gorm:
+    timeouts:
+      query: 60
+      transaction: 120
+      users:
```

Comment:
dont put here, we are putting under `app.security`

Reply:
Fixed